support/cas-server-support-themes-collection/src/main/resources/static/themes/twbs/js/cas.js

```diff
@@ -0,0 +1,121 @@
+/* global trackGeoLocation, jqueryReady */
+
+/* exported resourceLoadedSuccessfully */
+
+function requestGeoPosition() {
```

Comment:
Can you indicate why the theme needs this file? This is a lot of duplicate code that would somewhat quickly grow old.

Reply:
Resolved by moving material specific code to a separate file.